src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -785,7 +785,7 @@ public IQ handleColibriConferenceIQ(
             {
                 String channelID = channelIQ.getID();
                 int channelExpire = channelIQ.getExpire();
-                String channelBundleId = channelIQ.getChannelBundleId();
+                String channelBundleId = channelIQ.getEndpoint();
```

Comment:
We already have an "endpoint" variable in the same block, with the same value. Can we remove "channelBundleId" and use "endpoint" instead (we can move its initialization to the top of the block).